.github/workflows/build-go.yaml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches:
+      - main
+      - rc
+      - hotfix-rc
+
+  pull_request:
+
+jobs:
+  build:
+    name: Build
+    needs:
+      - build_rust
+      - generate-schemas
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup Go environment
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
❓  How does `${{ env.GO_VERSION }}` get it's value?

Reply:
Fixed.